test/functional/mempool_rebroadcast.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python3
+# Copyright (c) 2009-2019 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test mempool rebroadcast logic.
+
+"""
+
+from collections import defaultdict
+from decimal import Decimal
+from test_framework.mininode import P2PInterface, mininode_lock
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.messages import (
+        msg_mempool,
+        msg_getdata,
+        CInv
+)
+from test_framework.util import (
+        assert_equal,
+        assert_greater_than,
+        assert_greater_than_or_equal,
+        wait_until,
+        disconnect_nodes,
+        connect_nodes,
+        random_transaction,
+        gen_return_txouts,
+        create_confirmed_utxos,
+        create_lots_of_big_transactions,
+)
+import time
+import random
```

Comment:
Looks like a few of these imports (`Decimal`, `random`, ...) are unused.

Reply:
oh thanks ! do you have a tool to catch unused imports, or did you just happen to notice ?

Reply:
oh thanks ! do you have a tool to catch unused imports, or did you just happen to notice ?